modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/loader/XmlExtensionLoaderDelegate.java

```diff
@@ -80,9 +81,17 @@
   private static final Map<String, MetadataType> defaultOutputTypes = getCommonTypesBuilder()
       .put("void", typeLoader.load(Void.class))
       .build();
+  private static final Map<String, ParameterRole> parameterRoleTypes = ImmutableMap.<String, ParameterRole>builder()
```

Comment:
Create an static method to get the enum from an string representation instead of doing this.